src/components/timers/Countdown.tsx

```diff
@@ -1,3 +1,101 @@
-const Countdown = () => null;
+import { useState } from "react";
+import type { ChangeEvent } from "react";
+import { calculateSeconds } from "../../utils/helpers";
+import Time from "../generic/DisplayTime";
+import { Buttons } from "../generic/Button";
+import type { buttonInfo } from "../generic/Button";
+
+const interval = { intervalId: 0, isRunning: false };
+const initialState = { seconds: 0, minutes: 0 };
+
+const Countdown = () => {
+  const [currSeconds, setSeconds] = useState(0);
+  const setInputTime = (event: ChangeEvent<HTMLInputElement>) => {
+    if (event.target.id === "numMinutes") {
```

Comment:
dont rely on dom attributes for business logic. you can move this logic to the onChange handler in the inputs